From 301aee01eeef09855f43f0dc1b8aa0ed09f0330e Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Fri, 31 Mar 2017 14:50:54 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3368 emmc add property
 'mmc-hs200-1_8v'

Add property 'mmc-hs200-1_8v' for emmc, to support hs200, and
remove the unused property caps2-mmc-hs200.

Change-Id: I9040d004bf78987d5bed7e49ad891c47a47970a9
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3368-evb.dtsi    | 2 +-
 arch/arm64/boot/dts/rockchip/rk3368-geekbox.dts | 2 +-
 arch/arm64/boot/dts/rockchip/rk3368-p9.dts      | 2 +-
 arch/arm64/boot/dts/rockchip/rk3368-r88.dts     | 2 +-
 arch/arm64/boot/dts/rockchip/rk3368-sheep.dts   | 2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368-evb.dtsi b/arch/arm64/boot/dts/rockchip/rk3368-evb.dtsi
index 3f080be9e603..e8e1665aafe7 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368-evb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3368-evb.dtsi
@@ -155,7 +155,7 @@
 	broken-cd;
 	bus-width = <8>;
 	cap-mmc-highspeed;
-	caps2-mmc-hs200;
+	mmc-hs200-1_8v;
 	disable-wp;
 	mmc-pwrseq = <&emmc_pwrseq>;
 	non-removable;
diff --git a/arch/arm64/boot/dts/rockchip/rk3368-geekbox.dts b/arch/arm64/boot/dts/rockchip/rk3368-geekbox.dts
index f307b043ebb5..2e47aa3e7270 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368-geekbox.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3368-geekbox.dts
@@ -114,7 +114,7 @@
 	status = "okay";
 	bus-width = <8>;
 	cap-mmc-highspeed;
-	caps2-mmc-hs200;
+	mmc-hs200-1_8v;
 	clock-frequency = <150000000>;
 	disable-wp;
 	keep-power-in-suspend;
diff --git a/arch/arm64/boot/dts/rockchip/rk3368-p9.dts b/arch/arm64/boot/dts/rockchip/rk3368-p9.dts
index 11c88b502c47..f194a8106304 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368-p9.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3368-p9.dts
@@ -242,7 +242,7 @@
 &emmc {
 	bus-width = <8>;
 	cap-mmc-highspeed;
-	caps2-mmc-hs200;
+	mmc-hs200-1_8v;
 	supports-emmc;
 	disable-wp;
 	non-removable;
diff --git a/arch/arm64/boot/dts/rockchip/rk3368-r88.dts b/arch/arm64/boot/dts/rockchip/rk3368-r88.dts
index fff672ffa448..c871d70d3a35 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368-r88.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3368-r88.dts
@@ -188,7 +188,7 @@
 	broken-cd;
 	bus-width = <8>;
 	cap-mmc-highspeed;
-	caps2-mmc-hs200;
+	mmc-hs200-1_8v;
 	disable-wp;
 	mmc-pwrseq = <&emmc_pwrseq>;
 	non-removable;
diff --git a/arch/arm64/boot/dts/rockchip/rk3368-sheep.dts b/arch/arm64/boot/dts/rockchip/rk3368-sheep.dts
index 0b13e24e1b53..a65d2e92607a 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368-sheep.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3368-sheep.dts
@@ -184,7 +184,7 @@
 	status = "okay";
 	bus-width = <8>;
 	cap-mmc-highspeed;
-	caps2-mmc-hs200;
+	mmc-hs200-1_8v;
 	supports-emmc;
 	disable-wp;
 	non-removable;
-- 
2.35.3

